WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridFragment.java

```diff
@@ -525,6 +525,10 @@ public void search(String searchTerm) {
         mSearchTerm = searchTerm;
         List<MediaModel> mediaList = getFilteredMedia();
         mGridAdapter.setMediaList(mediaList);
+
+        if(isEmpty()){
```

Comment:
To keep the code style consistent, we should change `if(isEmpty()){` to `if (isEmpty()) {`.

Reply:
Sorry about this. I didn't check before push to GitHub 😬